tests/integration/test_cases/summary.py

```
@@ -112,15 +115,143 @@ async def test_fetch_summary(
                 logger.error(f"❌ Error in fetch summary test: {e}")
                 return False
 
+    async def test_summary_status_tracking(
+        self, collection_name: str, filename: str
+    ) -> bool:
+        """Test summary status tracking with polling (PENDING -> IN_PROGRESS -> SUCCESS)"""
+        async with aiohttp.ClientSession() as session:
+            try:
+                logger.info(f"📊 Testing status tracking for file: {filename}")
```

Ruff (0.14.5)

Logging statement uses f-string

(G004)